hubclient/errors.go

```diff
@@ -15,15 +15,19 @@
 package hubclient
 
 import (
-	"github.com/juju/errors"
+	"github.com/pkg/errors"
 )
 
 type HubClientError struct {
-	errors.Err
+	Err error
```

Comment:
Why not just embed error like juju.error was embedded?

Reply:
I did not want to change interface too much, since I don't know who referencing it from outside.
I will take a look

Reply:
changing to error will make it inaccessible outside (lowercase) not sure if this is a desired outcome. We can change it later needed.